flask_appbuilder/api/convert.py

```diff
@@ -167,6 +167,10 @@ def _column2field(self, datamodel, column, nested=True, enum_dump_by_name=False)
             field = EnumField(enum_class, dump_by=enum_dump_by, required=required)
             field.unique = datamodel.is_unique(column.data)
             return field
+        # is custom property method field?
+        if hasattr(getattr(_model, column.data), 'fget'):
+            return fields.Str(dump_only=True)
```

Comment:
@dpgaspar can we change this to `fields.Raw` instead?

I noticed that in Superset we're returning strings for properties of the database, including for booleans. This means that `allows_subquery` (for example) could potentially be returned as the string "false", which is truthy in Javascript.

I tested with `Raw` here, and I noticed it returns `false` instead of `"false"`, as expected, but I'm unsure if it would have other implications.